CIPs/cip_0047.md

```diff
@@ -0,0 +1,61 @@
+---
+cip: 47
+title: Round Change Timeout Formula Modification
+author: Joshua Gutow <@trianglesphere>
+discussions-to: https://github.com/celo-org/celo-proposals/issues/279
+status: Draft
+type: Standards Track, Ring 0
+created: 8/23/21
+license: Apache 2.0
+---
+
+## Simple Summary
+Add five seconds to the round change timeout for rounds greater or equal to 1.
+
+## Abstract
+Add five seconds to the round change timeout for rounds greater or equal to 1. This modifies the timeout schedule to make the timeouts for round 1 & 2 be longer than round 0.
+
+
+## Motivation
+The round change timeout is the amount of time that a validator node waits to before moving onto the next round in IBFT. This only occurs if a block is not finalized prior to the timeout. This timeout is exponential to enable nodes with differing clocks and poor network connections to be able to stabilize on the same round. In cases where the block is not able to be finalized in round 0, rounds 1 and 2 have a lower timeout meaning that the block is unlikely to finalized until round 3.
+
+## Specification
+```
+0 = baseTimeout + block time
+n = baseTimeout + block time + 2^n * backoff factor
+```
```

Comment:
Would like this to be more specific:

Round 0 timeout = baseTimeout + block time
Round n(n>0) timeout = baseTimeout + block time + 2^n * backoff factor